posthog/api/test/test_event_definition.py

```diff
@@ -118,3 +142,63 @@ def test_query_event_definitions(self):
         self.assertEqual(response.json()["count"], 1)
         for item in response.json()["results"]:
             self.assertIn(item["name"], ["watched_movie"])
+
+
+def create_organization(name: str) -> Organization:
+    return Organization.objects.create(name=name)
+
+
+def create_user(email: str, password: str, organization: Organization):
+    return User.objects.create_and_join(organization, email, password)
+
+
+def create_team(organization: Organization) -> Team:
```

Comment:
Followup: could consider moving this to some test utility directory and reuse across other postgres tests

Reply:
Yes I’m thinking we either have corresponding tests for endpoints or admin actions that are responsible for these actions, in which we can define these canonical helper functions